Sming/compiler/ld/eagle.app.v6.cpp.include.ld

```diff
@@ -0,0 +1,217 @@
+/*  
```

Comment:
Dear @anakod, thank you very much for the hint! I will try this as soon as I'm back on my desk on Friday.